.github/workflows/release.yaml

```diff
@@ -18,7 +18,6 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
-          cache: 'pip'
```

Comment:
This could potentially introduce bugs across runs that would be very difficult to debug while having very low impact since the run only takes a couple of minutes without it.